Docs/user_guide/get_help.md

```diff
@@ -0,0 +1,31 @@
+# Get Help
+
+[Contact the Slicer community or commercial partners](about.html#contact-us) if you have any questions, bug reports, or enhancement requests.
+
+## I need help in using Slicer
+
+- I have a question about how to use Slicer for a particular task or I'm not sure things are working right:
```

Comment:
`I'm not sure things are working right` -> `I'm not sure things are working correctly`